src/components/select-button/SelectButton.js

```diff
@@ -2,19 +2,21 @@
 import * as React from 'react';
 import classNames from 'classnames';
 
+import type { Position } from '../tooltip';
 import Tooltip from '../tooltip';
 import './SelectButton.scss';
 
 type Props = {
     children?: React.Node,
     className: string,
     error?: React.Node,
+    errorTooltipPosition?: Position,
     isDisabled: boolean,
 };
 
 const SelectButton = React.forwardRef<Props, HTMLButtonElement>(
-    ({ children, className = '', error, isDisabled = false, ...rest }: Props, ref) => (
-        <Tooltip isShown={!!error} position="middle-right" text={error} theme="error">
+    ({ children, className = '', error, errorTooltipPosition, isDisabled = false, ...rest }: Props, ref) => (
+        <Tooltip isShown={!!error} position={errorTooltipPosition || 'middle-right'} text={error} theme="error">
```

Comment:
I see we do `position={errorTooltipPosition || 'middle-right'}` in other places as well but i'm curious why not we don't default `errorTooltipPosition` in the function parameters

Reply:
No reason in particular, I see it both ways in the code. I like the default function parameter suggestion!